src/bepasty/utils/date_funcs.py

```diff
@@ -10,7 +11,10 @@ def get_maxlife(data, underscore):
     unit = data.get(unit_key, unit_default).upper()
     value_key = 'maxlife_value' if underscore else 'maxlife-value'
     value_default = '1'
-    value = int(data.get(value_key, value_default))
+    try:
+        value = int(data.get(value_key, value_default))
+    except (ValueError, TypeError):
+        raise BadRequest(description='Maxlife-Value header is incorrect')
```

Comment:
guess this should use value_key to generate the description.

Reply:
fixed as following

```python
        field = 'maxlife_value' if underscore else 'Maxlife-Value'
        raise BadRequest(description='{} header is incorrect'.format(field))
```